lib/modules/orangered.js

```diff
@@ -130,16 +130,27 @@ export function updateFromPage(doc?: HTMLElement) {
 	setUnreadCount(getUnreadCount(doc));
 }
 
-const _setUnreadCount = multicast(count => {
-	count = count || 0;
-
-	updateFaviconBadge(count);
-	updateTitle(count);
-	updateInboxElements(count);
-	updateMailCountElements(count);
-
-	if (count > 0) {
-		window.dispatchEvent(new Event('orangered'));
+const _setUnreadCount = multicast(data => {
+	const [
+		messageCount,
+		notificationCount,
+	] = data;
+
+	const sumCount = messageCount + notificationCount;
+
+	updateFaviconBadge(sumCount);
+	updateTitle(sumCount);
+	updateInboxElements(data);
+	updateMailCountElements(data);
+
+	if (sumCount > 0) {
+		const event = new Event('orangered');
+		event.data = {
```

Comment:
Event objects aren't shared, so this won't do anything. (Flow is MVP 😉)

This is what you want:
`const event = new CustomEvent('orangered', { detail: { sumCount, messageCount, notificationCount } });` 
(Chrome clones the object and Firefox drops it--FF only allows `detail` to be a primitive type, nothing we can do)